public/index.html

```diff
@@ -124,21 +124,23 @@ <h2>Kjellerstyret høst 2017</h2>
         <p>Hele kjellerstyret kan nås via <a href="mailto:ks@cyb.no">ks@cyb.no</a>.</p>
         <dl class="page-dl">
           <dt>Kjellermogul</dt>
-          <dd><a href="mailto:kjellermogul@cyb.no">Karl Hole Totland</a></dd>
+          <dd><a href="mailto:kjellermogul@cyb.no">Adrian Helle</a></dd>
           <dt>Barsjef</dt>
-          <dd><a href="mailto:cyb-barsjef@cyb.no">Lilly Arstad Helmersen</a></dd>
+          <dd><a href="mailto:cyb-barsjef@cyb.no">Marie Reichelt</a></dd>
           <dt>Kafésjef</dt>
-          <dd><a href="mailto:kafeansvarlig@cyb.no">Trine Høyås</a></dd>
+          <dd><a href="mailto:kafeansvarlig@cyb.no">Håkon Carlsen</a></dd>
           <dt>Økonomiansvarlig</dt>
-          <dd><a href="mailto:kasserer@cyb.no">Svein Manum</a></dd>
+          <dd><a href="mailto:kasserer@cyb.no">Maela Le Borgne</a></dd>
           <dt>Innkjøpsansvarlig</dt>
           <dd><a href="mailto:innkjopsansvarlig@cyb.no">Tor-Aksel Solberg</a></dd>
           <dt>Teknisk sjef</dt>
-          <dd><a href="mailto:tekniskansvarlig@cyb.no">Nicolas Harlem Eide</a></dd>
+          <dd><a href="mailto:tekniskansvarlig@cyb.no">Sindre Brurberg</a></dd>
           <dt>Utlånsansvarlig</dt>
           <dd><a href="mailto:kjeller-utlan@cyb.no">Benjamin Breiby</a></dd>
           <dt>DJ-sjef</dt>
-          <dd><a href="mailto:dj-sjef@cyb.no">Vincent Olsen</a></dd>
+          <dd><a href="mailto:dj-sjef@cyb.no">Nicolas Harlem Eide</a></dd>
+	  <dt>Arrangementskordinator</dt>
+	  <dd><a href="mailto:root@localhost">Edgar Andorsen</a></dd>
```

Comment:
Laget liste enda? (burde forresten bruke example.org for placeholder ting)

Reply:
Nope, tenkte å lage en ny en før eller siden